src/System.Private.ServiceModel/tests/Common/Scenarios/project.json

```diff
@@ -1,10 +1,14 @@
 {
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.1-rc3-23809",
-    "System.Collections.Concurrent": "4.0.0",
-    "System.Net.Http": "4.0.1-rc3-23809",
+	"System.Collections.Concurrent": "4.0.0",
+	"System.ComponentModel.EventBasedAsync": "4.0.10",
+	"System.Net.Http": "4.0.1-rc3-23809",
```

Comment:
Right, I want to make sure we actually don't need 4.0.1-* S.N.HTTP since we are downgrading the dependency of it in S.P.SM.